test/SourceKit/InterfaceGen/gen_swift_module_cross_import.swift.A.response

```diff
@@ -0,0 +1,313 @@
+import SwiftOnoneSupport
+
+public func fromA()
+
+
+// MARK: - B Additions
+
+import B
+
+// Available when B is imported with A
```

Comment:
Doesn't this seem a bit annoying to repeat it for every declaration? What do you think about only mention this once at the `MARK:` line?

Reply:
@brentdax asked for it on every declaration in the radar so I'll redirect the question there. Brent, are you okay with Argyrios' suggestion?
It looks repetitive to me, but I'm also not sure how likely users are to scroll up in larger overlays if they get to one of these symbols by jump-to-definition or cmd+f.

Reply:
> but I'm also not sure how likely users are to scroll up in larger overlays if they get to one of these symbols by jump-to-definition or cmd+f.
This is a good point, I retract my suggestion.